src/translator_testing_model/datamodel/pydanticmodel.py

```diff
@@ -558,6 +487,7 @@ class StandardsComplianceTestSuite(TestSuite):
     name: Optional[str] = Field(None, description="""A human-readable name for a Test Entity""")
     description: Optional[str] = Field(None, description="""A human-readable description for a Test Entity""")
     tags: Optional[List[str]] = Field(default_factory=list, description="""A human-readable tags for categorical memberships of a TestEntity (preferably a URI or CURIE). Typically used to aggregate instances of TestEntity into formally typed or ad hoc lists.""")
+    test_runner_settings: Optional[List[str]] = Field(default_factory=list, description="""Scalar parameters for the TestRunner processing a given TestEntity.""")
     
 
 class OneHopTestSuite(TestSuite):
```

Comment:
Does the OneHopTestSuite not need a TestCase class?

Reply:
Same for TRAPIValidation

Reply:
@maximusunc, isn't that covered by the class attribute:

```
 test_cases: Optional[Dict[str, TestCase]] = Field(default_factory=dict, description="""List of explicitly enumerated Test Cases.""")
```

That being said, I'm less convinced that these subclasses of **TestSuite** - i.e.  **AcceptanceTestSuite**, **BenchmarkTestSuite** (this one looks weird...), **StandardsComplianceTestSuite** and **OneHopTestSuite** - are even necessary (anymore, if they ever were) and could perhaps be removed from the model.  @sierra-moxon .. any thoughts about this?

Reply:
I think we use TestCase for the runner to help collate assets into a single query.  Happy to talk this through on a call.

Reply:
I'm fine with removing these subclass Suite classes. In the Test Harness, I basically start with grabbing a single TestSuite, and then for each TestCase, I look at the specified test_case_objective to determine which Runner to send it to.
I would also argue that a Suite can have different types of TestCases within it, so these subclass Suites might not even actually fit well.